phone/app/src/main/java/jp/ac/mayoi/maigocompass/RootScreen.kt

```diff
@@ -0,0 +1,75 @@
+package jp.ac.mayoi.maigocompass
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import jp.ac.mayoi.core.navigation.MemoryNavigation
+import jp.ac.mayoi.core.navigation.RankingNavigation
+import jp.ac.mayoi.core.navigation.TravelingNavigation
+import jp.ac.mayoi.core.resource.MaigoCompassTheme
+import jp.ac.mayoi.maigocompass.bottomNavigation.BottomNavItem
+
+@Composable
+fun RootScreen(){
+    val navController = rememberNavController()
+    val navBackStackEntry by navController.currentBackStackEntryAsState()
+    val currentDestination = navBackStackEntry?.destination?.hierarchy
+
+    Scaffold(
+        bottomBar = {
+            NavigationBar {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceAround
+                ) {
+                    BottomNavItem(
+                        label = "旅をする",
+                        resId = R.drawable.navbaritem_travel,
+                        isSelected = currentDestination?.any{
+                            it.route == TravelingNavigation::class.qualifiedName
+                        } == true,
+                        onClick = { navController.navigate(TravelingNavigation)}
+                    )
+                    BottomNavItem(
+                        label = "ランキング",
+                        resId = R.drawable.navbaritem_ranking,
+                        isSelected = currentDestination?.any{
+                            it.route == RankingNavigation::class.qualifiedName
+                        } == true,
+                        onClick = {navController.navigate(RankingNavigation)}
+                    )
+                    BottomNavItem(
+                        label = "おもいで",
+                        resId = R.drawable.navbaritem_memory,
+                        isSelected = currentDestination?.any{
+                            it.route == MemoryNavigation::class.qualifiedName
+                        } == true,
+                        onClick = {navController.navigate(MemoryNavigation)}
+                    )
+                }
+            }
+        }
+    ) { innerPadding ->
+        PhoneNavHost(
+            navController = navController,
+            innerPadding = innerPadding,
+        )
+    }
+}
+
+@Preview
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/fd31e7f306bbb6cb35174ceecb2a983d64de68e0/phone/app/src/main/java/jp/ac/mayoi/maigocompass/RootScreen.kt#L69"></span>Composables annotated with @Preview that are used only for previewing the UI should not be public.See https://slackhq.github.io/compose-lints/rules/#preview-composables-should-not-be-public for more information.</td>
    </tr>
  </tbody>
</table>